Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM ruby:2.5-alpine
 ARG BUNDLE_INSTALL_CMD
 
+ENV S3_PUBLISHED_LOCATIONS_IPS_BUCKET 'stub-bucket'
+ENV S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY 'stub-key'
+ENV ACCESS_KEY_ID 'stub-access-key'
+ENV SECRET_ACCESS_KEY 'stub-secret-access-key'
```

Comment:
This isn't required, just stub it out for the particular test that needs it I would say.

Reply:
(ACCESS_KEY_ID and SECRET_ACCESS_KEY) that is